Update 74_HusqvarnaAutomower.pm

fis1497 opened the pull request:
Moin,
hab ein paar Werte mit dazu genommen und
Kleine Reparaturen vorgenommen.

so wirkliches testen ist aktuell nicht drin :-)
und die Doku ist noch nicht nachgezogen

Gruß
Falk

fis1497 committed:
Update 74_HusqvarnaAutomower.pm

krannich commented:
Hi Falk,

vielen Dank. Ich prüfe das mal und werde es dann in FHEM Repo ziehen.
Wenn die Änderungen ersichtlich sind, kann ich die Doku auch nachziehen.

Viele Grüße
Dennis


Von meinem iPad gesendet

> Am 05.01.2020 um 14:57 schrieb fis1497 <notifications@github.com>:
> 
> ﻿
> Moin,
> hab ein paar Werte mit dazu genommen und
> Kleine Reparaturen vorgenommen.
> 
> so wirkliches testen ist aktuell nicht drin :-)
> und die Doku ist noch nicht nachgezogen
> 
> Gruß
> Falk
> 
> You can view, comment on, or merge this pull request online at:
> 
>   https://github.com/krannich/dkFHEM/pull/3
> 
> Commit Summary
> 
> Update 74_HusqvarnaAutomower.pm
> File Changes
> 
> M FHEM/74_HusqvarnaAutomower.pm (457)
> Patch Links:
> 
> https://github.com/krannich/dkFHEM/pull/3.patch
> https://github.com/krannich/dkFHEM/pull/3.diff
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub, or unsubscribe.

fis1497 committed:
Update 74_HusqvarnaAutomower.pm

fis1497 committed:
Update 74_HusqvarnaAutomower.pm

-nicht genutzte variablen erstmal auskommentiert.
--war beim testen aktuell nichts sinnvolles zu entlocken

-internal Timer startete nicht mehr bei DNS Problemen
--es wurde ein Remove timer ausgeführt und zu dem neuen timer kam das script nicht mehr

ReismannRS commented:
Kannst du die vier Timeouts noch von 5 auf 10 hochsetzen? Ansonsten ist die chance relativ hoch, dass das Script mit einem Error aussteigt und manuel neu gestartet werden muss. Mit einem Wert von 5 passiert das bei mir spätestens nach 2 Tagen irgendwann mal. Nach hochsetzen auf 10 ist dieses Problem nie wieder aufgetreten.

fis1497 committed:
Update 74_HusqvarnaAutomower.pm

ja Tim du hast recht mit "timeout 10" ist die Sache stabiler, fahre ich auch schon eine ganze Weile. Manchmal ist die API sehr träge.

In dem Zuge noch das Error Handling mit rein genommen , denn ich hatte zuerst nicht die langsamen abrufe im Verdacht sondern ein Token Problem.
Wird aber nun erstmal nicht benötigt.

ReismannRS commented:
Danke für´s einpflegen. Läuft alles super soweit. Ich habe allerdings noch ein Problem, für das ich noch keine Lösung gefunden habe. Wenn ich FHEM neu starte steht bei dem Husqvarna Modul immer "error" im status. Erst wenn ich ein "set myMower update" gesendet habe, verbindet er sich. Hast du dafür vielleicht eine Lösung?

fis1497 commented:
@ReismannRS da hilft nur "verbose" hoch auf 3 und mal in die Log schauen wo es klemmt
